src/components/course/CoursePage.jsx

```diff
@@ -139,16 +132,14 @@ const CoursePage = () => {
       if (isLoadingAny || !courseData || !courseRecommendations) {
         return undefined;
       }
-
       const {
         courseDetails,
         userEnrollments,
         userEntitlements,
         catalog,
       } = courseData;
-
       const { allRecommendations, samePartnerRecommendations } = courseRecommendations;
-
+      const courseEntitlementProductSku = findHighestLevelEntitlementSku(courseDetails.entitlements);
```

Comment:
[context] Moving this up into `CoursePage` such that it can be exposed via `CourseContext`.